code/game/objects/structures/signs/_signs.dm

```diff
@@ -30,111 +36,133 @@
 		return
 	user.examinate(src)
 
-/obj/structure/sign/attackby(obj/item/I, mob/user, params)
-	if(I.tool_behaviour == TOOL_WRENCH && buildable_sign)
-		user.visible_message("<span class='notice'>[user] starts removing [src]...</span>", \
-							 "<span class='notice'>You start unfastening [src].</span>")
-		I.play_tool_sound(src)
-		if(I.use_tool(src, user, 40))
-			playsound(src, 'sound/items/deconstruct.ogg', 50, TRUE)
-			user.visible_message("<span class='notice'>[user] unfastens [src].</span>", \
-								 "<span class='notice'>You unfasten [src].</span>")
-			var/obj/item/sign_backing/SB = new (get_turf(user))
+///This is a global list of all signs you can change an existing sign or new sign backing to, when using a pen on them.
+GLOBAL_VAR(editable_sign_types)
```

Comment:
put this in _globalvars and use GLOBAL_LIST_EMPTY

Reply:
Done.